mds/authent/migrations/0006_post_fill_allowed_scopes_with_default_values.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.7 on 2019-11-26 09:42
+
+from django.db import migrations
+
+
+def fill_default_scopes(apps, schema_editor):
+    Application = apps.get_model("authent", "Application")
+
+    for application in Application.objects.all():
+        application.users_allowed_scopes = application.scopes.copy()
+        application.save()
+
+
+class Migration(migrations.Migration):
+
```

Comment:
atomic = False (@bors-ltd ?)

Reply:
Better:
```python
Application.objects.update(users_allowed_scopes=F('scopes'))
```
Or maybe even better after my first point above, when `users_allowed_scopes` is null, fallback on `scopes`